hazelcast/src/main/java/com/hazelcast/config/alto/AltoConfig.java

```diff
@@ -1,15 +1,12 @@
 package com.hazelcast.config.alto;
 
-import com.hazelcast.config.InvalidConfigurationException;
-
 import java.util.Objects;
 
-public class AltoConfig {
-    private static final boolean DEFAULT_ENABLED = false;
-    private static final int DEFAULT_EVENTLOOP_COUNT = Runtime.getRuntime().availableProcessors();
+import static com.hazelcast.internal.util.Preconditions.checkPositive;
 
-    private boolean enabled = DEFAULT_ENABLED;
-    private int eventloopCount = DEFAULT_EVENTLOOP_COUNT;
+public class AltoConfig {
```

Comment:
Would be good to add some documentation, along with the beta annotation

Reply:
https://github.com/hazelcast/hazelcast/pull/23319/commits/57f4a326887f98dfdb6329505b31d651c2376e7c